src/test/java/teammates/test/cases/webapi/DeleteFeedbackSessionActionTest.java

```diff
@@ -23,13 +28,83 @@ protected String getRequestMethod() {
     @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        ______TS("Not enough parameters");
+
+        verifyHttpParameterFailure();
```

Comment:
You should test for failures for when there is only COURSE_ID provided or if there is only FEEDBACK_SESSION_NAME provided.